### PR TITLE
Visitor can create a review

### PR DESCRIPTION
**Pull request by Carrosen**
# [PT Story](https://www.pivotaltracker.com/story/show/166150749)

## Changes we propose in this PR
* User can click on an article on review page and be directed to the full article with review form including score and comment
* Sets reviewForm as a state in fullArticle component to be shown if article is unpublished

## What we have learned
* Usage of props and states

Screenshot from 2019-06-03 12-11-01

-----

Screenshot from 2019-06-03 12-11-16

-----

Screenshot from 2019-06-03 12-11-51

### CODE REVIEW

**Comment by raptorf1** [comment]
Look fine to me ;)

File: src/Components/ListOfUnpublishedArticles.jsx [hunk 1]
```diff
@@ -34,9 +35,18 @@ class ListOfUnpublishedArticles extends Component {
       </div>
     )
 
+    let message
+
+    if(this.state.review_success_message){
+      message = (
+        <p>Review successfully something</p>
```
Comment (raptorf1) [inline src/Components/ListOfUnpublishedArticles.jsx]: what is `something`????

File: src/index.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ import * as serviceWorker from './serviceWorker';
 import { BrowserRouter } from 'react-router-dom'
 import axios from 'axios'
 
-axios.defaults.baseURL = 'https://glocal-news.herokuapp.com'
+axios.defaults.baseURL = 'http://localhost:3002'
```
Comment (raptorf1) [inline src/index.js]: Dont't forget to change it back to heroku :)

**Review by raptorf1 — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 2 above)
Comment (Carrosen) [reply src/index.js]: done

File: src/Components/ListOfUnpublishedArticles.jsx (hunk 1 above)
Comment (Carrosen) [reply src/Components/ListOfUnpublishedArticles.jsx]: I have no clue, changed

File: cypress/fixtures/list_of_articles_with_published.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    "ingress": "How to spend 24 hours in Rome",
+    "body": "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem. Ut enim ad minima veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur? Quis autem vel eum iure reprehenderit qui in ea voluptate velit esse quam nihil molestiae consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur?",
+    "written_by": "George",
+    "created_at": "2019-05-24T09:56:01.168Z",
+    "image": "https://image.freepik.com/free-photo/roman-coliseum-seen-from-afar_1219-15.jpg",
+    "published": true,
+    "country": "Italy",
+    "city": "Rome" 
+  }
```
Comment (leiter007) [inline cypress/fixtures/list_of_articles_with_published.json:37]: Shouldn't we try to incorporate this mock-file in the list_of_articles fixture... So we don't have duplicates?

**Review by leiter007 — COMMENTED** [review]
Looks great! Only comment about the fixture file..

File: cypress/fixtures/list_of_articles_with_published.json (hunk 3 above)
Comment (Carrosen) [reply cypress/fixtures/list_of_articles_with_published.json:37]: I've tried that, the test doesnt pass, I have no idea why it doesnt work

**Pull request merged**